sync/landing.py

```diff
@@ -319,8 +319,18 @@ def has_metadata(self, sync):
     def add_metadata(self, sync):
         if self.has_metadata(sync):
             return
-        if sync.metadata_commit:
-            self.gecko_worktree.get().git.cherry_pick(sync.metadata_commit.sha1)
+        if sync.metadata_commit and not sync.metadata_commit.is_empty():
+            worktree = self.gecko_worktree.get()
+            try:
+                worktree.git.cherry_pick(sync.metadata_commit.sha1)
+            except git.GitCommandError as e:
+                if (str(e.status) == "1" and
```

Comment:
what does `"1"` mean? Can we avoid a magic number here?

Reply:
It's the status code that git exits with in this case. I don't think there's a reasonable way to avoid a magic number; definine `GIT_CHERRY_PICK_EXIT_CODE_ON_EMPTY_COMMIT` doesn't seem helpful. But I could add a comment.

Reply:
Let's add a comment just so that we know what it means in the future